lms/djangoapps/teams/static/teams/js/collections/team.js

```diff
@@ -3,6 +3,8 @@
     define(['teams/js/collections/base', 'teams/js/models/team', 'gettext'],
         function(BaseCollection, TeamModel, gettext) {
             var TeamCollection = BaseCollection.extend({
+                sortField: 'name',
```

Comment:
I would argue that we should change the API too. Why would it default to a non-useful sort order?